__init__.py

```diff
@@ -57,164 +94,92 @@ def defaultTrigger(self):
         return "jb "
 
     def initialize(self):
-        pass
+        plugin_dir = os.path.dirname(__file__)
+        self.editors = [
+            Editor(
+                name="Android Studio",
+                icon=plugin_dir + "/androidstudio.svg",
+                config_dir_prefix="Google/AndroidStudio",
+                binaries=["studio", "androidstudio"]),
+            Editor(
+                name="CLion",
+                icon=plugin_dir + "/clion.svg",
+                config_dir_prefix="JetBrains/CLion",
+                binaries=["clion"]),
+            Editor(
+                name="DataGrip",
+                icon=plugin_dir + "/datagrip.svg",
+                config_dir_prefix="JetBrains/DataGrip",
+                binaries=["datagrip"]),
+            Editor(
+                name="DataSpell",
+                icon=plugin_dir + "/dataspell.svg",
+                config_dir_prefix="JetBrains/DataSpell",
+                binaries=["dataspell"]),
+            Editor(
+                name="GoLand",
+                icon=plugin_dir + "/goland.svg",
+                config_dir_prefix="JetBrains/GoLand",
+                binaries=["goland"]),
+            Editor(
+                name="IntelliJ IDEA",
+                icon=plugin_dir + "/idea.svg",
+                config_dir_prefix="JetBrains/IntelliJIdea",
+                binaries=["idea"]),
```

Comment:
the old list was a bit bigger and had more entries here. I compiled them mostly from the archlinux aur which has a huge variety of possible binary names (see where they are linked into the /usr/bin directory here: https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=intellij-idea-ultimate-edition#n60 , https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=intellij-idea-community-edition-no-jre#n34 , https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=intellij-idea-ue-eap#n34 , https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=intellij-idea-community-edition-git#n68 . maybe even idea.sh (including the `.sh`) would make some sense. I don't even know what other linux distros (eg. Ubuntu ppa's) name their binaries. I think for intellij, at least `intellij-idea-ultimate-edition`, `idea-ce` and `idea-ue-eap` should be added (after going through all archlinux packages).

Reply:
Done!

I did not find any "idea.sh" (as often the binary becomes "idea", or "intellij-long-name-edition")